src/gp/inc/GPMSA.h

```diff
@@ -364,13 +364,16 @@ class GPMSAFactory
   // All the GP priors information for a scalar GP follows:
   void setUpHyperpriors();
 
+  // Number of dimensions preserved from the SVD of simulation outputs
+  unsigned int num_svd_terms;
+
   // Domains for all the hyperpriors
   typename ScopedPtr<VectorSpace<V, M> >::Type oneDSpace;
 
-  // Emulator mean
-  typename ScopedPtr<V>::Type emulatorMeanMin;
-  typename ScopedPtr<V>::Type emulatorMeanMax;
-  typename ScopedPtr<BoxSubset<V, M> >::Type emulatorMeanDomain;
+  // Truncation error precision
+  typename ScopedPtr<V>::Type truncationErrorPrecisionMin;
+  typename ScopedPtr<V>::Type truncationErrorPrecisionMax;
+  typename ScopedPtr<BoxSubset<V, M> >::Type truncationErrorPrecisionDomain;
```

Comment:
It's the error term for when we use fewer components of the SVD than we have outputs for.

I probably shouldn't have mixed it in this PR; I was originally intending to make more sweeping changes at once but then when I realized the regression tests would have to change I decided to "checkpoint" much earlier.